go/coordinator/internal/logservice/grpc/record_log_service.go

```diff
@@ -0,0 +1,47 @@
+package grpc
+
+import (
+	"context"
+	"github.com/chroma/chroma-coordinator/internal/grpcutils"
+	"github.com/chroma/chroma-coordinator/internal/proto/logservicepb"
+	"github.com/chroma/chroma-coordinator/internal/types"
+	"github.com/pingcap/log"
+	"go.uber.org/zap"
+	"google.golang.org/protobuf/proto"
+)
+
+func (s *Server) PushLogs(ctx context.Context, req *logservicepb.PushLogsRequest) (*logservicepb.PushLogsResponse, error) {
+	res := &logservicepb.PushLogsResponse{}
+	collectionID, err := types.ToUniqueID(&req.CollectionId)
+	if err != nil || collectionID == types.NilUniqueID() {
+		log.Error("collection id format error", zap.String("collection.id", req.CollectionId))
+		grpcError, err := grpcutils.BuildInvalidArgumentGrpcError("collection_id", "wrong collection_id format")
+		if err != nil {
+			log.Error("error building grpc error", zap.Error(err))
+			return nil, err
+		}
+		return nil, grpcError
```

Comment:
I think we should use named return params. To avoid shadowing, and this kind of return